File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Xrts/XRT0201U00GrdRadioListItemInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Xrts
{
    [Serializable]
	public class XRT0201U00GrdRadioListItemInfo
	{
		private String _bunho;
		private String _orderDate;
		private String _fkxrt0201;
		private String _xrayCode;
		private String _xrayName;
		private String _xrayGubun;
		private String _xrayCodeIdx;
		private String _xrayCodeIdxNm;
		private String _tubeVol;
		private String _tubeCur;
		private String _xrayTime;
		private String _tubeCurTime;
		private String _irradiationTime;
		private String _xrayDistance;
		private String _dataRowState;

		public String Bunho
		{
			get { return this._bunho; }
			set { this._bunho = value; }
		}

		public String OrderDate
		{
			get { return this._orderDate; }
			set { this._orderDate = value; }
		}

		public String Fkxrt0201
		{
			get { return this._fkxrt0201; }
			set { this._fkxrt0201 = value; }
		}

		public String XrayCode
		{
			get { return this._xrayCode; }
			set { this._xrayCode = value; }
		}

		public String XrayName
		{
			get { return this._xrayName; }
			set { this._xrayName = value; }
		}

		public String XrayGubun
		{
			get { return this._xrayGubun; }
			set { this._xrayGubun = value; }
		}

		public String XrayCodeIdx
		{
			get { return this._xrayCodeIdx; }
			set { this._xrayCodeIdx = value; }
		}

		public String XrayCodeIdxNm
		{
			get { return this._xrayCodeIdxNm; }
			set { this._xrayCodeIdxNm = value; }
		}

		public String TubeVol
		{
			get { return this._tubeVol; }
			set { this._tubeVol = value; }
		}

		public String TubeCur
		{
			get { return this._tubeCur; }
			set { this._tubeCur = value; }
		}

		public String XrayTime
		{
			get { return this._xrayTime; }
			set { this._xrayTime = value; }
		}

		public String TubeCurTime
		{
			get { return this._tubeCurTime; }
			set { this._tubeCurTime = value; }
		}

		public String IrradiationTime
		{
			get { return this._irradiationTime; }
			set { this._irradiationTime = value; }
		}

		public String XrayDistance
		{
			get { return this._xrayDistance; }
			set { this._xrayDistance = value; }
		}

		public String DataRowState
		{
			get { return this._dataRowState; }
			set { this._dataRowState = value; }
		}

		public XRT0201U00GrdRadioListItemInfo() { }

		public XRT0201U00GrdRadioListItemInfo(String bunho, String orderDate, String fkxrt0201, String xrayCode, String xrayName, String xrayGubun, String xrayCodeIdx, String xrayCodeIdxNm, String tubeVol, String tubeCur, String xrayTime, String tubeCurTime, String irradiationTime, String xrayDistance, String dataRowState)
		{
			this._bunho = bunho;
			this._orderDate = orderDate;
			this._fkxrt0201 = fkxrt0201;
			this._xrayCode = xrayCode;
			this._xrayName = xrayName;
			this._xrayGubun = xrayGubun;
			this._xrayCodeIdx = xrayCodeIdx;
			this._xrayCodeIdxNm = xrayCodeIdxNm;
			this._tubeVol = tubeVol;
			this._tubeCur = tubeCur;
			this._xrayTime = xrayTime;
			this._tubeCurTime = tubeCurTime;
			this._irradiationTime = irradiationTime;
			this._xrayDistance = xrayDistance;
			this._dataRowState = dataRowState;
		}

	}
}

File: MED-CLOUD/med-data/src/main/java/nta/med/data/dao/medi/xrt/XRT0201U00GrdRadioListItemInfo.java
package nta.med.data.dao.medi.xrt;

import java.util.Date;

public class XRT0201U00GrdRadioListItemInfo {
	private String bunho ;
	private Date orderDate ;
	private Double fkxrt0201 ;
	private String xrayCode ;
	private String xrayName ;
	private String xrayGubun ;
	private String xrayCodeIdx ;
	private String xrayCodeIdxNm ;
	private String tubeVol ;
	private String tubeCur ;
	private String xrayTime ;
	private String tubeCurTime;
	private String irradiationTime ;
	private String xrayDistance ;
	public XRT0201U00GrdRadioListItemInfo(String bunho, Date orderDate,
			Double fkxrt0201, String xrayCode, String xrayName,
			String xrayGubun, String xrayCodeIdx, String xrayCodeIdxNm,
			String tubeVol, String tubeCur, String xrayTime,
			String tubeCurTime, String irradiationTime, String xrayDistance) {
		super();
		this.bunho = bunho;
		this.orderDate = orderDate;
		this.fkxrt0201 = fkxrt0201;
		this.xrayCode = xrayCode;
		this.xrayName = xrayName;
		this.xrayGubun = xrayGubun;
		this.xrayCodeIdx = xrayCodeIdx;
		this.xrayCodeIdxNm = xrayCodeIdxNm;
		this.tubeVol = tubeVol;
		this.tubeCur = tubeCur;
		this.xrayTime = xrayTime;
		this.tubeCurTime = tubeCurTime;
		this.irradiationTime = irradiationTime;
		this.xrayDistance = xrayDistance;
	}
	public String getBunho() {
		return bunho;
	}
	public void setBunho(String bunho) {
		this.bunho = bunho;
	}
	public Date getOrderDate() {
		return orderDate;
	}
	public void setOrderDate(Date orderDate) {
		this.orderDate = orderDate;
	}
	public Double getFkxrt0201() {
		return fkxrt0201;
	}
	public void setFkxrt0201(Double fkxrt0201) {
		this.fkxrt0201 = fkxrt0201;
	}
	public String getXrayCode() {
		return xrayCode;
	}
	public void setXrayCode(String xrayCode) {
		this.xrayCode = xrayCode;
	}
	public String getXrayName() {
		return xrayName;
	}
	public void setXrayName(String xrayName) {
		this.xrayName = xrayName;
	}
	public String getXrayGubun() {
		return xrayGubun;
	}
	public void setXrayGubun(String xrayGubun) {
		this.xrayGubun = xrayGubun;
	}
	public String getXrayCodeIdx() {
		return xrayCodeIdx;
	}
	public void setXrayCodeIdx(String xrayCodeIdx) {
		this.xrayCodeIdx = xrayCodeIdx;
	}
	public String getXrayCodeIdxNm() {
		return xrayCodeIdxNm;
	}
	public void setXrayCodeIdxNm(String xrayCodeIdxNm) {
		this.xrayCodeIdxNm = xrayCodeIdxNm;
	}
	public String getTubeVol() {
		return tubeVol;
	}
	public void setTubeVol(String tubeVol) {
		this.tubeVol = tubeVol;
	}
	public String getTubeCur() {
		return tubeCur;
	}
	public void setTubeCur(String tubeCur) {
		this.tubeCur = tubeCur;
	}
	public String getXrayTime() {
		return xrayTime;
	}
	public void setXrayTime(String xrayTime) {
		this.xrayTime = xrayTime;
	}
	public String getTubeCurTime() {
		return tubeCurTime;
	}
	public void setTubeCurTime(String tubeCurTime) {
		this.tubeCurTime = tubeCurTime;
	}
	public String getIrradiationTime() {
		return irradiationTime;
	}
	public void setIrradiationTime(String irradiationTime) {
		this.irradiationTime = irradiationTime;
	}
	public String getXrayDistance() {
		return xrayDistance;
	}
	public void setXrayDistance(String xrayDistance) {
		this.xrayDistance = xrayDistance;
	}

}
